linchpin/InventoryFilters/InventoryFilter.py

```diff
@@ -109,3 +109,45 @@ def add_common_vars(self, host_groups, layout):
                     else:
                         host_string += " " + var + "=" + common_vars[var]
                 self.config.set(group, host_string)
+
+    def get_hostname(self, data, cfgs, default_fields):
+        if '__IP__' in cfgs.keys():
+            val = self.config_value_helper(data, cfgs['__IP__'])
+            if val:
+                return (cfgs['__IP__'], val)
+        for var in default_fields:
+            val = self.config_value_helper(data, var)
+            if val:
+                return (var, val)
+        return ''
+
+    def set_config_values(self, host_data, instance, cfgs={}):
+        """
+        finds the key associated with each config variable and get the
+        value
+        """
+        if cfgs is None:
+            print "cfgs is None"
```

Comment:
Print statements are not python3 compatible
Please use
print() instead
@14rcole

Reply:
That was a debugging statement anyway.  Fixed now